src/components/Navigation/Sections/FilterBar.js

```diff
@@ -10,45 +10,83 @@ export default class FilterBar extends Component {
     activeFilters: PropTypes.object,
     onFilter: PropTypes.func,
     onSort: PropTypes.func,
-    hasComponents: PropTypes.bool
+    hasComponents: PropTypes.bool,
+    showSortFilter: PropTypes.bool,
+    showLicenseFilter: PropTypes.bool,
+    showSourceFilter: PropTypes.bool,
+    showReleaseDateFilter: PropTypes.bool,
+    customLicenses: PropTypes.object,
```

Comment:
should this be `customLicenses: PropTypes:Array` ?

Reply:
Right, i will fix it, thanks!